src/main/java/org/broadinstitute/consent/http/models/Institution.java

```diff
@@ -36,24 +41,36 @@ public Institution(Integer id, String name, String itDirectorName, String itDire
     this.updateUser = updateUser;
   }
 
-  public void setName(String name, Integer updateUser) {
```

Comment:
remove the update user parameter from all set methods and add set methods for update user, create user, create date, and update date

Reply:
this follows the structure of other model classes and relies on the user to update user and update date when they make a change instead of doing it automatically